app/assets/javascripts/admin/utils/directives/variant_autocomplete.js.coffee

```diff
@@ -22,6 +22,7 @@ angular.module("admin.utils").directive "variantAutocomplete", ($timeout) ->
               q: term
               distributor_id: scope.distributor_id
               order_cycle_id: scope.order_cycle_id
+              available_for_subscriptions: scope.available_for_subscriptions
```

Comment:
related to comment below: is there a way we can make this subscriptions change without injecting subscriptions logic into this core part of ofn?
here, I'd prefer to see a specific subs_variant_auto_complete than seeing subscriptions in this base one.